getting-found-staying-found/en/getting-found-visibility.md

```diff
@@ -275,6 +275,26 @@ It is increasingly common that funders require projects to include a knowledge t
 * Social media can amplify your story, but the substantive content usually resides elsewhere such as a news website, institutional blog, or your journal’s announcements page.
 * Journal content can have a long life. Consider promoting articles published several years earlier, especially if an article can fill an information vacuum for something current.
 
+## Checklist 
+
+> Contributed by Stuart Yeates and Max Sullivan
+
+Before you start any promotion:
+* Validate your OAI-PMH feed. There are a number of OAI-PMH validators which provide a third-party check of you metadata, for example: https://validator.oaipmh.com/ https://oval.base-search.net/ https://validator.dini.de/ 
+* Check your SSL certificate using one of a number of checkers, for example: https://www.ssllabs.com/ssltest/analyze.html https://www.sslchecker.com/sslchecker
```

Comment:
For any hyperlinks in documentation, please create descriptive links rather than adding raw links. You can read more [in our style guide](https://docs.pkp.sfu.ca/contributing/en/style-and-format#accessible-hyperlinks). In Markdown, the following formatting should be used:

`[SSL Labs](https://www.ssllabs.com/ssltest/analyze.html)`

There are multiple throughout the PR that should be converted to descriptive links.